edgelet/edgelet-core/src/module.rs

```diff
@@ -181,6 +181,7 @@ where
 /// time, but for now, this will have to do...
 pub trait NestedEdgeBodge {
     fn parent_hostname_resolve_image(&mut self, parent_hostname: &str);
+    fn parent_hostname_resolve_auth(&mut self, parent_hostname: &str);
```

Comment:
Just rename the existing method to `parent_hostname_resolve` and have it fixup both the image and the auth.

Reply:
Done!